Library/Formula/hugo.rb

```diff
@@ -105,11 +105,22 @@ class Hugo < Formula
     url "https://github.com/yosssi/ace.git",
         :revision => "1f82044938a7180f6fb6bbb3a29688d1e6dbe74b"
   end
-  go_resource "gopkg.in/fsnotify.v0" do
-    url "https://github.com/go-fsnotify/fsnotify.git",
+
+  # Install correct version of fsnotify needed by stable/head
+  stable do
+    go_resource "gopkg.in/fsnotify.v0" do
+      url "https://github.com/go-fsnotify/fsnotify.git",
         :branch => "v0",
         :revision => "ea925a0a47d225b2ca7f9932b01d2ed4f3ec74f6"
+    end
   end
+  head do
+    go_resource "gopkg.in/fsnotify.v1" do
+      url "https://github.com/go-fsnotify/fsnotify.git",
+        :revision => "6549b98005f3e4026ad9f50ef7d5011f40ba1397"
+    end
+  end
```

Comment:
I think it's more readable to have the `go_resource` definition here, together with the others, and wrapped in the corresponding `stable` / `head` blocks (which are ultimately just fancy conditional blocks, so they work fine), rather than artificially move this one to the top where the other `stable` block is, just for the sake of having a single one. But I'll leave it up to the maintainers to make a final decision.